kanidmd/src/lib/core/https.rs

```diff
@@ -1278,6 +1307,11 @@ pub fn create_https_server(
     account_route
         .at("/:id/_credential/:cid/_lock")
         .get(do_nothing);
+    account_route
+        .at("/:id/_credential/backup_code")
```

Comment:
These should be under :cid or `.at("/:id/_credential/primary")`. This is for future multiple credential handling

Reply:
In the future there is a plan to have multiple credentials per account, but it's not fully fleshed out, but we want the routes to reflect this possible change for now.